Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx

```diff
@@ -0,0 +1,55 @@
+import AlertBox from "../AlertBox/AlertBox";
+import SectionText from "../SectionText/SectionText";
+import SectionTitle from "../SectionTitle/SectionTitle";
+import "./ContentCenter.css";
+
+const ContentCenter = () => {
+  return (
+    <div style={{ display: "flex", justifyContent:"space-between" }} className="content-center">
+      <div style={{ display: "flex", flexDirection: "column", justifyContent: "space-between", alignItems:"flex-start"}}>
+        <AlertBox text={"Exclusive news here"} />
+        <div>
+          <SectionTitle title={"ADELE"} />
+          <SectionText
+            text={
+              "She is just one of the most important singer in the world currently "
+            }
+          />
+        </div>
+
+        <div >
+          <SectionTitle size={80} title={"The Cyrus's siblings "} />
+          <SectionText
+            text={
+              "The news about the new tour by Miley Cyrus and her what her family think about it "
+            }
+          />
+        </div>
+
+        <div >
+          <SectionTitle size={80} title={"DEMI LOVATO"} />
+          <SectionText
+            text={
+              "The news about the new tour by Miley Cyrus and her what her family think about it "
+            }
+          />
+        </div>
+      </div>
+
+
+      <div className="right" style={{  marginTop:"300px",height:"100%", display: "flex", flexDirection: "column", justifyContent: "center", alignItems: "center"}}>
```

Comment:
@jhonmartinezG , using inline styles is a bad practice please move this to class, also this div should be a section

Reply:
fixed in the new commit